code/cutscene/ffmpeg/FFMPEGDecoder.cpp

```diff
@@ -165,7 +169,7 @@ std::unique_ptr<DecoderStatus> initializeStatus(std::unique_ptr<InputStream>& st
 	if (audioStream < 0) {
 		if (audioStream == AVERROR_STREAM_NOT_FOUND) {
 			mprintf(("FFmpeg: No audio stream found in file!\n"));
-		} else if (videoStream == AVERROR_DECODER_NOT_FOUND) {
+		} else if (audioStream == AVERROR_DECODER_NOT_FOUND) {
```

Comment:
It looks like this is a bugfix.  Is there a corresponding Mantis or GitHub issue, or HLP post, or is this just something you noticed?

Reply:
It's a simple copy&paste error I noticed while editing the code. It's located inside a code block that is only used for error handling and it doesn't even change the end result so there is no reason to fix this in a separate PR.

Reply:
I didn't mean to imply that this should be a separate PR, just wondering if it addressed an existing ticket.